spec/system/claims/schools/users/view_users_of_a_school_spec.rb

```diff
@@ -1,38 +1,94 @@
 require "rails_helper"
 
-RSpec.describe "View users for school", service: :claims, type: :system do
-  let(:school) { create(:school, :claims, urn: "123456") }
+RSpec.describe "Claims support user view users for school", service: :claims, type: :system do
+  scenario do
+    given_a_school_exists_with_claims_users
+    and_i_am_signed_in_as_a_support_user
 
-  before do
-    setup_school_and_anne_membership
-  end
+    when_i_select_organisations
+    then_i_see_the_organisations_page
+
+    when_i_click_on_the_school
```

Comment:
```suggestion
    when_i_click_on_springfield_elementary
```
Please be more descriptive as to which school you are clicking on